zipline/utils/run_algo.py

```diff
@@ -155,15 +156,20 @@ def choose_loader(column):
         env = TradingEnvironment(environ=environ)
         choose_loader = None
 
+    if not trading_calendar:
+        trading_calendar = get_calendar('NYSE')
```

Comment:
Both `TradingAlgorithm` and `SimulationParameters` take a `TradingCalendar` object so I added those lines to get the calendar object specifically for `TradingAlgorithm`, IIRC

If anything, I'm thinking I could just remove the handling from `create_simulation_parameters, because I'm just using the same object here for both trading algo and simparams.

Thoughts?

Reply:
Ah, makes sense that we need to pass an actual `TradingCalendar` when constructing the `TradingAlgorithm`. Seems like we're using `create_simulation_parameters` a bunch of places in the tests where we don't pass a calendar, so probably fine to leave this as-is